source/people/main.rst

```diff
@@ -91,6 +91,10 @@ Undergraduate Students & Interns
 |                                    |                                    |
 | :doc:`Ellis Wilson <ejwilson3>`    |  :doc:`Alex Swenson <aaswenson>`   | 
 +------------------------------------+------------------------------------+
+|                                    |                                    |
```

Comment:
This is OK, but adding another column might make more sense than adding another row.